src/FloatingButton/FloatingButton.tsx

```diff
@@ -0,0 +1,199 @@
+import * as classnames from "classnames";
+import * as PropTypes from "prop-types";
+import * as React from "react";
+
+import { Button } from "../Button/Button";
+import FlexBox from "../flex/FlexBox";
+
+import "./FloatingButton.less";
+
+const cssClass = {
+  ACTIVE: "FloatingButton--active",
+  inactive: positionY => `FloatingButton--inactive--${positionY}`,
+
+  propStyle: prop => `FloatingButton--${prop}`,
+
+  BUTTON: "FloatingButton--button",
+  CONTAINER: "FloatingButton--container",
+  REMOVE: "FloatingButton--remove",
+  SUB_ELEMENT: "FloatingButton--subElement",
+};
+
+const propTypes = {
+  animate: PropTypes.bool,
+
+  additionalButtons: PropTypes.array,
+
+  bgColor: PropTypes.string,
+  className: PropTypes.string,
+  closeLabel: PropTypes.node,
+  label: PropTypes.node,
+
+  onClick: PropTypes.func,
+
+  offsetX: PropTypes.number,
+  offsetY: PropTypes.number,
```

Comment:
Why do we have offsets if the developer positions the menu with screen locations `positionX` and `positionY` ("top", "left", "bottom", "right")?

Reply:
It's still useful to be able to move the button away from the default offset from the edge of the screen (for example if they want to place it low enough to be below a top bar)